src/components/victory-voronoi.jsx

```diff
@@ -123,7 +125,8 @@ VictoryVoronoi.propTypes = {
   svgStyles: React.PropTypes.object,
   svgWidth: React.PropTypes.number,
   triangle: React.PropTypes.bool,
-  velocity: React.PropTypes.number
+  velocity: React.PropTypes.number,
+  animate: React.PropTypes.bool
```

Comment:
I think the convention in other Victory is to have `animate` be an object (with stuff like `velocity` in it), which gets passed to `VictoryAnimation`. If no such object is passed (defaults to undefined), skip `VictoryAnimation`.